### ISSUE TITLE
Support print request info of  "cy.route()". 

### ISSUE THREAD

**Issue by JFRabbit** [archfz/cypress-terminal-report#25]
In my case:
```
cy.server()
cy.route('POST', '**/upload**').as('upload')
cy.customCommandUpload('...')
cy.wait('@upload')
```

Sometimes I got below error: "token is invalid", and need to check token of request header.

<img width="835" alt="截屏2020-05-21上午10 34 45" src="https://user-images.githubusercontent.com/22906174/82517120-f944a000-9b4e-11ea-8c66-b6883cf2cb50.png">

I hope this plugin support print some request info, include header and payload.

**Comment by archfz** [comment]
This would be a new feature. Currently we do not support logging the request headers. What we would need for this is probably some kind of flag to say what headers to log.

**Comment by archfz** [comment]
Released in 1.3.0.
